x-pack/plugins/canvas/public/renderers/advanced_filter/component/advanced_filter.js

```diff
@@ -7,7 +7,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
-import './advanced_filter.scss';
```

Comment:
What's going on with these style changes? Do these maybe need to be added to `public/style/index.scss`? Same goes for all the other styles that were removed.